docs/modules/ROOT/pages/references/crds.adoc

```diff
@@ -642,10 +642,10 @@ VSHNDBaaSBackupSpec contains settings to control the backups of an instance.
 [cols="25a,75a", options="header"]
 |===
 | Field | Description
-| *`timeOfDay`* __string__ | TimeOfDay for doing daily backups, in UTC. Format: "hh:mm:ss".
+| *`schedule`* __string__ | Schedule how often should backup be made, format: https://regex101.com/r/5Yz8e9/1, example: '30 23 * * *' - each day at 23:30
```

Comment:
Pointing to a regex does certainly not help to understand. Isn't this just normal cron syntax?

Reply:
they implement only numbers in their cron syntax, I'm afraid if we write it's `regular cron style` someone would try to use `30 4 * feb mon/fri` because, it's valid cron syntax.

This is why I decided to reference regex website, because the same regex is used to validate cronjob syntax in our component

Reply:
```suggestion
| *`schedule`* __string__ | Schedule how often should backup be made,example: '30 23 * * *' - each day at 23:30. Please note that this isn't a regular Cron syntax.
```

And somewhere add a more meaningful description in readable language what exactly is meant, a regex is not human-readable. Maybe the Stackgres docs has some help?